src/blocks/notmuch.rs

```diff
@@ -13,6 +13,7 @@
 //!
 //! Key | Values | Default
 //! ----|--------|--------
+//! `format` | A string to customise the output of this block. See below for available placeholders. | `" $icon{ $name|} $count "`
```

Comment:
Now name configuration option could be dropped? Format can be used.

Reply:
Yes :+1: